docs/configuration.md

```diff
@@ -32,6 +32,7 @@ initial-development: false  # OPTIONAL, defaults to `true`
 | `version-prefix` | `"*"` | An optional prefix specifying the tags to consider. "*" is a special value, meaning the closest version is used, regardless of prefix |
 | `version-scheme` | `semver` | The versioning scheme to use. Supported values: `semver`, `sdkver`. |
 | `release-branches` | `/^release\/.*\d+\.\d+\.*$/` | A regex specifying from which branch(es) only `PATCH` version bump are allowed. |
+| `release-discussion-category` | `undefined` | A string specifying an existing GitHub discussion category name. This requires GitHub discussions to be enabled on the repository. |
```

Comment:
In terms of errors reported in case enabled:
* Discussion disabled in repository: 
```
  Error: Unable to create release with the name "1.0.7" due to HTTP request error (status 404):
  Discussions are not enabled on this repository.
```
* No existing discussion category:
```
  Error: Unable to create release with the name "1.0.9" due to HTTP request error (status 404):
  Discussion could not be created. Make sure you passed a valid category name.
```